<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTableView" name="tableView"/>
        </item>
        <item>
         <widget class="QTreeWidget" name="treeWidget">
          <column>
           <property name="text">
            <string>分数段</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>数学</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>语文</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>英语</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>平均分</string>
           </property>
          </column>
          <item>
           <property name="text">
            <string>60分以下</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>60~69</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>70~79</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>80~89</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>90以上</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>平均分</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>最高分</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>最低分</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="icon">
         <iconset>
          <normalon>:/images/BarChart.png</normalon>
         </iconset>
        </attribute>
        <attribute name="title">
         <string>BarChart</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0" colspan="2">
          <widget class="QChartView" name="chartViewBar"/>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="btn_flashBar">
           <property name="text">
            <string>刷新柱状图</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>177</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="icon">
         <iconset>
          <normalon>:/images/PieChart.png</normalon>
         </iconset>
        </attribute>
        <attribute name="title">
         <string>PieChart</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="3">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>HoleSize</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="btn_flashPie">
           <property name="text">
            <string>刷新饼图</string>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_pieSize">
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>21</height>
            </size>
           </property>
           <property name="minimum">
            <double>0.000000000000000</double>
           </property>
           <property name="value">
            <double>0.700000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="8">
          <widget class="QComboBox" name="comboBox_them">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>主题1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>主题2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>主题3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>主题4</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>PieSize</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_holeSize">
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>21</height>
            </size>
           </property>
           <property name="value">
            <double>0.200000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>分析数据</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBox_course">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>语文</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>数学</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>英语</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="7">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>主题</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="9">
          <widget class="QChartView" name="chartViewPie"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="icon">
         <iconset>
          <normalon>:/images/StackedBar.png</normalon>
         </iconset>
        </attribute>
        <attribute name="title">
         <string>StackedBar</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QPushButton" name="btn_flashStackedBar">
           <property name="text">
            <string>刷新堆叠柱状图</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>485</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QChartView" name="chartViewStackedBar"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_4">
        <attribute name="icon">
         <iconset>
          <normalon>:/images/PercentBar.png</normalon>
         </iconset>
        </attribute>
        <attribute name="title">
         <string>PerCentBar</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QPushButton" name="btn_flashPercentBar">
           <property name="text">
            <string>刷新百分比柱状图</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>473</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QChartView" name="chartViewPercentBar"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_5">
        <attribute name="icon">
         <iconset>
          <normalon>:/images/ScatterChart.png</normalon>
         </iconset>
        </attribute>
        <attribute name="title">
         <string>ScatterChart</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="QPushButton" name="btn_flashScatterChart">
           <property name="text">
            <string>刷新图标</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>497</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QChartView" name="chartViewScatterChart"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>888</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_init"/>
   <addaction name="separator"/>
   <addaction name="action_survey"/>
   <addaction name="separator"/>
   <addaction name="action_exit"/>
  </widget>
  <action name="action_init">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/init.png</normaloff>:/images/init.png</iconset>
   </property>
   <property name="text">
    <string>初始化数据</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="action_survey">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/survey.png</normaloff>:/images/survey.png</iconset>
   </property>
   <property name="text">
    <string>数据统计</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/exit.png</normaloff>:/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
